olm/src/commonTest/kotlin/io/github/matrixkt/olm/BaseTest.kt

```diff
@@ -0,0 +1,3 @@
+package io.github.matrixkt.olm
+
+interface BaseTest
```

Comment:
Is this interface strictly required? I think you could just call `runTest` as is.

Also take a look at [`BeforeTest`](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/-before-test/). It'll let you run `Olm.init()` without having to add `runTest` every single test.

Reply:
i've seen https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/-before-test/ and this did not work at all. I copied this way of testing from sqldeligtht which does it this way

Reply:
This might still be open.

Reply:
Fair enough. `BaseTest` can go though.

Reply:
no it cannot go, its the base for all tests and is extended with runTest function depending on the targets  like here `actual fun BaseTest.runTest(block: () -> Unit) { block() }`

Reply:
should i add a comment to the file or class?

Reply:
I still don't see the need for the base is what I'm getting at.
`runTest` can be a free function like so `actual fun runTest(block: () -> Unit) { block() }`, then the test classes don't need to extend an interface.

Reply:
now i get it. sorry yes this would remove the need for the empty interface.